src/helpers/message.ts

```diff
@@ -0,0 +1,125 @@
+import { IModify, IPersistence, IRead } from '@rocket.chat/apps-engine/definition/accessors';
+import { IRoom, RoomType } from '@rocket.chat/apps-engine/definition/rooms';
+import { IUser } from '@rocket.chat/apps-engine/definition/users';
+import { LayoutBlock } from '@rocket.chat/ui-kit';
+
+import { NotificationsController } from './notification';
+
+/**
+ * Gets a direct message room between bot and another user, creating if it doesn't exist
+ *
+ * @param read
+ * @param modify
+ * @param appUser
+ * @param username the username to create a direct with bot
+ * @returns the room or undefined if botUser or botUsername is not set
+ */
+export async function getDirect(
+  read: IRead,
```

Comment:
please create an object with the params of this function, it reduces the mental overhead